### ISSUE TITLE
Logs are not creating in the tcp host mode

### ISSUE THREAD

**Issue by pawansaket** [kylefarris/clamscan#61]
Hi,
I am trying to use the clamscan in the tcp mode and want to write the logs local file, but it works on local mode and not in the server  using the tcp mode but logs are not appearing in the log file.
my configuration is, I am trying but don't know what is going on.
![image](https://user-images.githubusercontent.com/26107608/94018023-ba10af00-fdcd-11ea-9a78-a34741c5e9f8.png)
![image](https://user-images.githubusercontent.com/26107608/94018031-bda43600-fdcd-11ea-8884-f3daff2f8ce4.png)

**Comment by kylefarris** [comment]
Please read the following section in the docs: [A note about using this module via sockets or TCP](https://github.com/kylefarris/clamscan#a-note-about-using-this-module-via-sockets-or-tcp) especially the section under "The following configuration items are not honored (unless the module falls back to binary method)". 

Notably, the bullet point:
 * `scan_log` - remote clamd service config will dictate this

In other words, you will not be able to see clamd logs on on your local server when the clamd service runs on a remote server. You would need to look at the logs on the server running the clamd service. Theoretically, you could run a syslog server on your local server that your remote server logs to but that's way outside the scope of this library.
